org-docs/PROJECT-CHARTER.md

```diff
@@ -0,0 +1,136 @@
+Similarly, Maintainers of a particular Project might need to coordinate with their counterpart on another to ensure the requirements for a new bug-fix or release are met, and to preempt dependency conflicts and incompatibility.
+
+**3.4. Consideration of Views and Objections**. Prompt consideration shall be given to the written views and objections of all Contributors.
+
+**3.5. Written procedures**. This governance document and other materials documenting this project's development process shall be available to any interested person.
+
+**3.6. Project's roadmap and release planning**.
+Maintainers shall keep open planification documents and roadmap definition and maintenance, and are responsible to seek feedback and engage contributors in the process of defining the future lines of the Project.
+Principal investigators will inform Maintainers about the existing commitments with funding agencies and other relevant institutions in terms of deliverables and milestones that the Project might need to observe to respond to such commitments.
+In case of conflict between the Maintainers' criteria and the Principal investigators' proposal, an appeal must be addressed to the Organization Steering Committee.
+The Project may maintain a `.maint/ROADMAP.md` stating how roadmaps are built, how issues are prioritized and organized, and the how the release process and achievement of deliverables and milestones are monitored.
+
+## 4. Release process and Scientific publication.
+
+### 4.1. Releases
+
+This Project follows the *NiPreps Community* guidelines for [releases](https://www.nipreps.org/devs/releases/) and [version synchrony](https://www.nipreps.org/devs/versions/).
+The release process may be initiated by any Maintainer, who will follow the prescribed documentation to the effect.
+This project has an automated deployment pipeline triggered with a git tagging operation by a Maintainer, which ensures the minting of the correct version number, python and docker packaging, publication of packages in open repositories and finally, the posting of a new release entry at Zenodo.
+
+**Long-term support (LTS) release series**.
+End-user applications may commit to extended windows of support for particular version series (see [related documentation here](https://www.nipreps.org/devs/releases/#long-term-support-series)).
+Past release series are maintained with *maintenance* branches named `maint/<YY>.<minor>.x` (for instance fMRIPrep's `maint/20.2.x`).
+For those maintenance branches tagged as LTS series, the `.maint/MAINTAINERS.md` and the `.maint/PIs.md` may differ from those of the development (called `master` or `main`) branch.
+New contributors to a maintenance branch will be added to the `.maint/CONTRIBUTORS.md` file of that branch, and then upstreamed to posterior `maint/<series>` branches and the `master`/`main` branch of the repository.
```

Comment:
This doesn't really fall under LTS.

Reply:
Not sure I understand what you mean.

Reply:
Okay, I think I finally understood - I moved this line with the description of contributors.

Reply:
All three lines highlighted are not specific to LTS, but any maintained series.